apis/placement/v1beta1/binding_types.go

```diff
@@ -123,14 +123,6 @@ const (
 	// - "Unknown" means it is unknown.
 	ResourceBindingOverridden ResourceBindingConditionType = "Overridden"
 
-	// ResourceBindingBound indicates the bound condition of the given resources.
-	// Its condition status can be one of the following:
-	// - "True" means the corresponding work CR is created in the target cluster's namespace.
-	// - "False" means the corresponding work CR is not created yet.
-	// - "Unknown" means it is unknown.
-	// TODO, will be replaced by "WorkSynchronized"
-	ResourceBindingBound ResourceBindingConditionType = "Bound"
```

Comment:
Hi Zhiying! Could you do the removal in the v1 package as well? I know it wouldn't affect the CRD, but a little bit consistency would still be better I think